src/main.js

```diff
@@ -100,6 +85,28 @@ utils.logger = winston.createLogger({
   ],
 });
 
+// config must be loaded after the logger is initialized
+config = loadConfig();
+
+if (config.discordLogging.active.toUpperCase() === "TRUE") {
```

Comment:
All references to config values need a null coalesce (`??`) with a default value that makes sense. This could be abstracted away with a function.